tests/serverpod_test_server/lib/src/protocol/user_not_found_exception.yaml

```diff
@@ -0,0 +1,3 @@
+exception: UserNotFound
```

Comment:
Instead of `UserNotFound` perhaps make it more descriptive for the tests. I.e. `ExceptionWithData` would be better.